Submit Assienment/Shubham_Shinde/preview.html

```diff
@@ -0,0 +1,61 @@
+ 
+
+ <!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Invitation Card</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css"
+        integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+<body>
+    <div class="background">
+        <div class="invitation-wrapper"> 
+            <!-- <img src="./img/1.png" alt=""> -->
+        <div class="invitation-background">
+            <div class="middle-light-blue"></div>
+            <div class="top-left-blue"></div>
+            <div class="top-right-yellow"></div>
+            <div class="bottom-left-yellow"></div>
+            <div class="yellow-dash"></div>
+            <div class="yellow-dash-bottom"></div>
+        </div>
+
+        <div class="invitation-content">
+            <div class="invitation-header"></div>
+                
+            <div class="invitation-body">
```

Comment:
don't add **invitation-body** as wrapper for content

Reply:
done